man/signal-cli.1.adoc

```diff
@@ -130,7 +130,7 @@ Uses a list of phone numbers to determine the statuses of those users. Shows if
 
 [NUMBER [NUMBER ...]]::
 One or more numbers to check.
-*--json*::
+*--j*, *--json*::
```

Comment:
Good idea. I've made an issue for it ( #407 ). Not sure how I'd get the global json-output mode and detailed mode to be mutually exclusive though.

Reply:
To clarify on the mutual exclusion thing; I thought it might be best for args to complain if you do both --output=json and -d at the same time since -d is always assumed. Its not a big issue though and help does state that it does -d by default for json which I think is also fine.

Reply:
yeah, I don't think it's an issue. There's no harm if the user specifies the -d option additionally. Describing the behavior in the help section is fine.

For the existing commands that support --json (at least for receive) I'd prefer the flag to keep working and log a deprecation warning. Then it can be completely removed in a later version.

Reply:
Sure, I'll add that in for the `receive` and `getProfileUpdate`

Reply:
You can use the logger to output the warning on stderr, so it doesn't mix with the json output on stdout :)

`logger.warn(...)`

Reply:
Done :)